otx/algorithms/common/configs/training_base.py

```diff
@@ -372,4 +372,20 @@ class BaseTilingParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.NONE,
         )
 
+        ir_scale_factor = configurable_float(
```

Comment:
```suggestion
        tiling_ir_scale_factor = configurable_float(
```

Reply:
The name `ir_scale_factor` is could be confusing. Since this variable is only used for tiling, I would suggest to change the name

Reply:
I agree also

Reply:
I didn't include the term "tiling" in the IR scale factor because while it's related to tiling, the concept primarily involves increasing the input shape of the IR model, and since it's under tiling group, it should be clear enough from my POV. Nevertheless, I understand it may cause confusion and I can make the change.